app/src/main/java/com/musicplayer/musicManagement/regularPlaylist/RemoveRegularPlaylist.kt

```diff
@@ -1,18 +1,21 @@
 package com.musicplayer.musicManagement.regularPlaylist
 
 import arrow.core.Either
+import arrow.core.Right
 import com.musicplayer.framework.messaging.Command
 import com.musicplayer.framework.messaging.CommandHandler
+import com.musicplayer.musicManagement.repositories.PlaylistRepository
 import com.musicplayer.framework.messaging.Error
 import java.util.*
 
 data class RemoveRegularPlaylist(
     val playlistId: UUID
 ) : Command
 
-class RemoveRegularPlaylistHandler() :
+class RemoveRegularPlaylistHandler(private val playlistRepository: PlaylistRepository) :
     CommandHandler<RemoveRegularPlaylist> {
-    override suspend fun handle(command: RemoveRegularPlaylist) : Either<Error, Unit> {
-        return Either.right(Unit)
+    override suspend fun handle(command: RemoveRegularPlaylist): Either<Error, Unit> {
+        playlistRepository.remove(command.playlistId)
```

Comment:
Please do check if given playlist is Main one and if so don't allow to remove it